.gitignore

```diff
@@ -3,5 +3,6 @@
 Gemfile.lock
 coverage
 gemfiles/*.lock
+gemfiles/vendor
```

Comment:
📝  [chore] I needed to run `BUNDLE_GEMFILE=gemfiles/rails_7.2.gemfile bundle` to debug test failures but it creates files under `gemfiles/vendor`. So I decided to let it ignore them.